### DISCUSSION TITLE
Error transcribing audio using tiny model

### DISCUSSION THREAD

**Discussion by mominurr** [openai/whisper#2440] [Q&A]
```
try:
    MODEL = whisper.load_model("tiny",download_root=r"D:\W.S-PROJECT\test\env\Lib\site-packages\whisper")
except Exception as e:
    LOG.error(f"Error loading model: {e}")
    pass

# Transcribe the audio using the tiny model
try:
    result = self.model.transcribe(path_to_mp3)
    key=result["text"].strip()
except Exception as e:
    LOG.error(f"Error transcribing audio using tiny model | error: {e}")
    key=""

```

When I am running my exe on Windows, I get this error:

log.pyc | 2024-11-14 21:04:31 | Error transcribing audio using tiny model | error: [Errno 2] No such file or directory: 'C:\\Users\\Mominur\\AppData\\Local\\Temp\\_MEI64962\\whisper\\assets\\mel_filters.npz'

Please can you help me to solve this issue?

**Comment by Bruxar** [comment]
Hello, I encountered a similar issue while trying to package my Python program using auto-py-to-exe. The root cause of the problem was that certain files, such as mel_filters.npz, were not being located correctly during runtime. These files were included in the build but placed in a directory structure that the program could not find.

To resolve this, I added the entire whisper folder from my .venv directory to the **--add-data** parameter. This ensured that the required files were bundled correctly and accessible from the appropriate directory during execution.

The PyInstaller command generated by auto-py-to-exe that successfully resolved the issue was:

`pyinstaller --noconfirm --onedir --console --add-data "project-location\.venv\Lib\site-packages\whisper;whisper/" "project-location\main.py"`

I hope this explanation helps resolve the issue for you. Best regards!
